go-controller/pkg/node/gateway_init_linux_test.go

```diff
@@ -109,6 +109,18 @@ func shareGatewayInterfaceTest(app *cli.App, testNS ns.NetNS,
 			Cmd:    "ovs-vsctl --timeout=15 get interface eth0 ofport",
 			Output: "7",
 		})
+		fexec.AddFakeCmd(&ovntest.ExpectedCmd{
+			Cmd:    "ip route replace table 7 172.16.1.0/24 via 10.1.1.1 dev ovn-k8s-mp0",
+			Output: "0",
+		})
```

Comment:
maybe a good idea running `ip route show table 7` to be sure its created

Reply:
hmm the problem is the test only reflects the commands called in the code base, so simply calling a show in actual code doesn't make sense, but the docs and tests confirm the creation of table else we would have erro-ed out when trying ip route replace table 7.